crates/router/src/connector/shift4/transformers.rs

```diff
@@ -212,10 +211,9 @@ impl TryFrom<&api_models::payments::WalletData> for Shift4PaymentMethod {
             | payments::WalletData::TouchNGoRedirect(_)
             | payments::WalletData::WeChatPayQr(_)
             | payments::WalletData::CashappQr(_)
-            | payments::WalletData::SwishQr(_) => Err(errors::ConnectorError::NotSupported {
-                message: utils::SELECTED_PAYMENT_METHOD.to_string(),
-                connector: "Shift4",
-            }
+            | payments::WalletData::SwishQr(_) => Err(errors::ConnectorError::NotImplemented(
```

Comment:
Here, use OR( | ) between `WeChatPayRedirect` and `AliPayQr` Payment methods as we have common error message now for all the Payment Methods.